packages/koa-bolt/src/index.ts

```diff
@@ -0,0 +1,36 @@
+import Router, { RouterParamContext } from '@koa/router';
+import { ExpressReceiver } from '@slack/bolt';
+import { Middleware, ParameterizedContext } from 'koa';
+import compose from 'koa-compose';
+
+const defaultEndpoints = {
+  events: '/slack/events',
+};
+
+export interface KoaBoltOptions {
+  receiver: ExpressReceiver;
+  endpoints?: Record<string, string>;
+}
+
+export default function koaBolt({
+  receiver,
+  endpoints = defaultEndpoints,
+}: KoaBoltOptions): Middleware<
+  ParameterizedContext<any, RouterParamContext>,
+  ParameterizedContext<any, RouterParamContext>
+> {
+  const boltRouter = new Router();
+
+  const boltMiddleware: Middleware = ctx => {
+    delete ctx.res.statusCode;
+    ctx.respond = false;
```

Comment:
https://koajs.com/#ctx-respond

> Note that using this is not supported by Koa. This may break intended functionality of Koa middleware and Koa itself. Using this property is considered a hack and is only a convenience to those wishing to use traditional `fn(req, res)` functions and middleware within Koa.

https://koajs.com/#ctx-res

> Bypassing Koa's response handling is not supported. Avoid using the following node properties: `res.statusCode` ...

I think we're okay with these, maybe worth flagging/noting somewhere? Maybe caveats in the readme

Reply:
Commented reasoning